documentation/User-Documentation/Objectives-List.md

```diff
@@ -2,9 +2,15 @@
 
 ## Action: `action`
 
-This objective completes when player clicks on given block type. This can be further limited by location condition and item in hand condition. First argument is type of the click, it can be right, left or any. Next is a `block selector`. You can also specify `loc:` argument, followed by standard location format and `range:` followed by a number (or variable). It will define where the clicked block needs to be, as opposed to "where you must be" in location condition. If you add argument `cancel`, the click will be canceled (chest will not open, button will not be pressed etc.)
+This objective completes when player clicks on given block type. This can be further limited by location condition and
```

Comment:
```suggestion
This objective completes when the player clicks on the given block type. This can be further limited by the location condition and
```